frontends/pytorch/csrc/CMakeLists.txt

```diff
@@ -19,22 +27,26 @@ else()
   )
 endif()
 
-add_library(_torch_mlir SHARED
+add_library(NPCOMPTorchMLIRExt SHARED
   init_python_bindings.cpp
   )
-set_target_properties(_torch_mlir PROPERTIES PREFIX "")
-get_property(mlir_libs GLOBAL PROPERTY MLIR_ALL_LIBS)
-target_link_libraries(_torch_mlir
-  NPCOMPATenDialect
+target_link_libraries(NPCOMPTorchMLIRExt
   ${TORCH_LIBRARIES}
-  ${mlir_libs}
   ${PYTHON_LIBRARIES}
   ${torch_mlir_optional_libraries}
   torch_python
+
+  # NPCOMP shared library.
+  NPCOMP
```

Comment:
The fact that we need a comment here explaining the target name suggests that it needs a better name.